hazelcast/config.py

```diff
@@ -706,6 +706,12 @@ class ClientProperties(object):
     Period in seconds to collect statistics.
     """
 
+    SERIALIZATION_INPUT_RETURNS_BYTEARRAY = ClientProperty("hazelcast.serialization.input.returns.bytearray", False)
+    """
+    Input#read_byte_array returns a List if property is True, otherwise it will return a byte-array.
```

Comment:
List if property is True -> List if property is False
Changing this to False -> Changing this to True